/find broken on staging

Reproduce:
Access https://staging.openoversight.com/find leave the forms blank and click through step 1 to 4. Clicking submit leads to "Internal Server Error".

The following is the error log from the staging server related to the error

```
Aug 25 01:32:52 staging gunicorn[25467]: [2020-08-25 01:32:52 +0000] [25483] [ERROR] Error handling request /find
Aug 25 01:32:52 staging gunicorn[25467]: Traceback (most recent call last):
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/gunicorn/workers/sync.py", line 135, in handle
Aug 25 01:32:52 staging gunicorn[25467]:     self.handle_request(listener, req, client, addr)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
Aug 25 01:32:52 staging gunicorn[25467]:     respiter = self.wsgi(environ, resp.start_response)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/app.py", line 2309, in __call__
Aug 25 01:32:52 staging gunicorn[25467]:     return self.wsgi_app(environ, start_response)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/app.py", line 2295, in wsgi_app
Aug 25 01:32:52 staging gunicorn[25467]:     response = self.handle_exception(e)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/app.py", line 1741, in handle_exception
Aug 25 01:32:52 staging gunicorn[25467]:     reraise(exc_type, exc_value, tb)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
Aug 25 01:32:52 staging gunicorn[25467]:     raise value
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/app.py", line 2292, in wsgi_app
Aug 25 01:32:52 staging gunicorn[25467]:     response = self.full_dispatch_request()
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/app.py", line 1815, in full_dispatch_request
Aug 25 01:32:52 staging gunicorn[25467]:     rv = self.handle_user_exception(e)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/app.py", line 1718, in handle_user_exception
Aug 25 01:32:52 staging gunicorn[25467]:     reraise(exc_type, exc_value, tb)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
Aug 25 01:32:52 staging gunicorn[25467]:     raise value
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
Aug 25 01:32:52 staging gunicorn[25467]:     rv = self.dispatch_request()
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
Aug 25 01:32:52 staging gunicorn[25467]:     return self.view_functions[rule.endpoint](**req.view_args)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/OpenOversight/OpenOversight/app/main/views.py", line 87, in get_officer
Aug 25 01:32:52 staging gunicorn[25467]:     if form.validate_on_submit():
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/flask_wtf/form.py", line 100, in validate_on_submit
Aug 25 01:32:52 staging gunicorn[25467]:     return self.is_submitted() and self.validate()
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/wtforms/form.py", line 310, in validate
Aug 25 01:32:52 staging gunicorn[25467]:     return super(Form, self).validate(extra)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/wtforms/form.py", line 152, in validate
Aug 25 01:32:52 staging gunicorn[25467]:     if not field.validate(self, extra):
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/wtforms/fields/core.py", line 195, in validate
Aug 25 01:32:52 staging gunicorn[25467]:     self.pre_validate(form)
Aug 25 01:32:52 staging gunicorn[25467]:   File "/home/nginx/oovirtenv/venv/lib/python3.5/site-packages/wtforms/fields/core.py", line 471, in pre_validate
Aug 25 01:32:52 staging gunicorn[25467]:     for v, _ in self.choices:
Aug 25 01:32:52 staging gunicorn[25467]: TypeError: 'NoneType' object is not iterable
```

--- a/OpenOversight/app/main/forms.py
+++ b/OpenOversight/app/main/forms.py
@@ -46,7 +46,7 @@ class FindOfficerForm(Form):
     unique_internal_identifier = StringField('unique_internal_identifier', default='', validators=[Regexp(r'\w*'), Length(max=55)])
     dept = QuerySelectField('dept', validators=[DataRequired()],
                             query_factory=dept_choices, get_label='name')
-    unit = SelectField('unit', default='Not Sure', validators=[Optional()])
+    unit = StringField('unit', default='Not Sure', validators=[Optional()])
     rank = StringField('rank', default='Not Sure', validators=[Optional()])  # Gets rewritten by Javascript
     race = SelectField('race', default='Not Sure', choices=RACE_CHOICES,
                        validators=[AnyOf(allowed_values(RACE_CHOICES))])

